mmv1/third_party/terraform/website/docs/r/sql_database_instance.html.markdown

```diff
@@ -244,6 +244,8 @@ The `settings` block supports:
     for more details and supported versions. Postgres supports only shared-core machine types,
     and custom machine types such as `db-custom-2-13312`. See the [Custom Machine Type Documentation](https://cloud.google.com/compute/docs/instances/creating-instance-with-custom-machine-type#create) to learn about specifying custom machine types.
 
+* `user_labels` - (Optional) A set of key/value user label pairs to assign to the instance.
+
 * `edition` - (Optional) The edition of the instance, can be `ENTERPRISE` or `ENTERPRISE_PLUS`.
 
 The optional `settings.advanced_machine_features` subblock supports:
```

Comment:
`settings.advanced_machine_features` being this high is probably adding to the confusion as well. It should probably go lower, probably right above the `settings.database_flags` block. Do you mind moving that ( and it's child `threads_per_core` field as well? @AmandaHassoun

Reply:
will do @c2thorn 👍🏼

Reply:
`settings.advanced_machine_features` only supports the `settings.advanced_machine_features.threads_per_core` field, but it makes it look like fields like `activation_policy` `availability_type`, etc are under it as well. I think that is maybe why you thought `settings.user_labels` is under `settings.advanced_machine_features`.

Line 283 is when we start listing all of the other parent fields with subfields, so we should move `settings.advanced_machine_features` somewhere down there. Right before `settings.database_flags` would be nice so that there is some alphabetical ordering.

Reply:
@c2thorn updated! Also, I have no access to the failures and they seem to be unrelated to my changes..

Reply:
I did totally misunderstand.. Fixed!

Reply:
To keep alphabetical ordering, I put `threads_per_core` before `time_zone`.